### DISCUSSION TITLE
Error: Input file contains unsupported image format

### DISCUSSION THREAD

**Discussion by TheZoker** [immich-app/immich#4206] [Q&A] [answered]
Hi there,

For some of my images, immrich is no able to generate a thumbnail. These are standard jpg images, so I'm not sure, what is going on. 

Here is a sample image:
![20230710_212532](https://github.com/immich-app/immich/assets/1368405/b9537147-b57e-452e-8604-ec9981edfd5a)

In the browser it looks like this:
![image](https://github.com/immich-app/immich/assets/1368405/f2acee62-f9d9-4ddb-835a-0f70ac66c907)

This is the error I get, when trying to generate a thumbnail
```
[Nest] 6  - 09/24/2023, 9:30:14 PM    WARN [MediaRepository] Could not determine colorspace of image, defaulting to p3 profile
[Nest] 6  - 09/24/2023, 9:30:14 PM   ERROR [JobService] Unable to run job handler (thumbnailGeneration/generate-jpeg-thumbnail): Error: Input file contains unsupported image format
[Nest] 6  - 09/24/2023, 9:30:14 PM   ERROR [JobService] Error: Input file contains unsupported image format
[Nest] 6  - 09/24/2023, 9:30:14 PM   ERROR [JobService] Object:
{
  "id": "75d5580c-83bc-4794-a79b-4333565a4564"
}
```

Also no meta data is displayed for this image. 

Other photos from the same camera (a Samsung S23+) work without any issue. 
All photos are added through a external library. 

Is there anything I can do to fix this? 

Thanks!

**Comment by TheZoker** [comment, marked as answer]
Seems like it was a permission issue. Solved it by changing some file permissions.

**Reply by illnesse** [reply]
Remember what you changed exactly? Seems like i have the same issue with a lxc mount, 777 doesn't seem to do the trick

**Reply by mfuellbier** [reply]
I would be interested as well

**Reply by vswraith** [reply]
@TheZoker I am also facing the same issue...what permissions need to change?
